plugins/woocommerce-blocks/assets/js/blocks/product-collection/tracks-utils.ts

```diff
@@ -40,6 +40,10 @@ export const useTracksLocation = ( templateSlug: string ) => {
 		return postType;
 	}
 
+	if ( ! templateSlug ) {
+		return Locations.OTHER;
```

Comment:
Not sure that this is correct, before merging the PR let's wait for @kmanijak and @imanish003 feedback.

Reply:
I see the other PR has some conflicts. I'll resolve them and merge the other PR to unlock you.

Reply:
Thanks, @kmanijak! 🚀 

I'm going to close this PR!